src/renderer/features/operation/init/ui/OperationFooter.tsx

```diff
@@ -23,12 +27,17 @@ export const OperationFooter = ({
   transaction,
   account,
   totalAccounts,
+  xcmConfig,
+  onXcmFeeChange,
   onDepositChange,
   onFeeChange,
   onFeeLoading,
 }: Props) => {
   const { t } = useI18n();
 
+  // TODO: Check why transaction can be empty
+  const isXcmTransfer = XcmTypes.includes(transaction?.type);
```

Comment:
Is this TODO supposed to be left for the feature? In that case its worth to create a separate task to not to forget about it